dashboard/app/views/levels/editors/_grid.html.haml

```diff
@@ -39,9 +43,10 @@
           return { tileType: cell };
         });
       });
+
       ReactDOM.render(React.createElement(dashboard.GridEditor, {
         serializedMaze: serializedMaze,
-        skin: '#{@level.skin === 'hoc2015x' ? 'starwarsgrid' : 'playlab'}',
+        skin: '#{skin}',
```

Comment:
moved to https://github.com/code-dot-org/code-dot-org/pull/16065/files#diff-f929033c0e44d0166de8dbb179628258R33